### PR TITLE
Fix iTunesConnect error when not setting changelog

### PR DESCRIPTION
**Pull request by mpirri**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

When running `pilot` without a changelog, feedback email, or app description and no beta information set on iTunesConnect, iTC was returning an error. This fixes that issue.

### CODE REVIEW

File: pilot/lib/pilot/build_manager.rb [hunk 1]
```diff
@@ -121,7 +121,7 @@ def describe_build(build)
     end
 
     def should_update_build_information(options)
```
Comment (KrauseFx) [inline pilot/lib/pilot/build_manager.rb]: If you have some extra minutes, maybe you can update the method name to follow the Ruby convention of ending with a question mark for methods that return boolean values => `should_update_build_information?` 👍

**Review by KrauseFx — APPROVED** [review]
Looks great, added one comment that would be a separate PR 👍

**Review by ohayon — APPROVED** [review]
I agree with @KrauseFx 's comment, but this is 💯  good find!

**Review by KrauseFx — APPROVED** [review]
🏣

**Pull request merged**